components/shared/modal/ErrorModal.js

```diff
@@ -19,7 +23,7 @@ const ErrorModal = ({ isVisible, closeModal }) => {
         <TouchableOpacity onPress={closeModal}>
           <Text>{$t('error.cancel')}</Text>
         </TouchableOpacity>
-        <TouchableOpacity onPress={() => NavigationService.navigate('AuthLoading')}>
+        <TouchableOpacity onPress={() => _restartApp()}>
```

Comment:
If the params are the same or none like in this case, there is no need to create another anonymous function, so for example, you can write
`onPress={data=>someFunc(data)}` as `onPress={someFunc}`
`onPress={()=>someFunc()}` as `onPress={someFunc}`

and in this case just `onPress={_restartApp}`

Reply:
Ah, Good catch. Thanks